config_flow.py

```
@@ -42,32 +59,108 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     async def async_step_user(
         self, user_input: dict[str, Any] | None = None
     ) -> FlowResult:
-        """Handle the initial step."""
+        """Handle the initial step - choose auth method."""
+        if user_input is not None:
+            self._auth_method = user_input[CONF_AUTH_METHOD]
+            self._host = user_input[CONF_HOST]
+            
+            if self._auth_method == AUTH_METHOD_ACCESS_CODE:
+                return await self.async_step_access_code()
+            else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)